pydis_site/apps/content/resources/guides/python-guides/windows/microsoft-store.md

```diff
@@ -1,36 +1,73 @@
 ---
-title: Why not to install Python from the Microsoft Store
-description: This guide explains the issues with using Python installed from the Microsoft Store
+title: Why Not to Install Python from the Microsoft Store
```

Comment:
```suggestion
title: Why **not** to Install Python from the Microsoft Store
```

Reply:
Or maybe even put it in full capital

Reply:
Markdown doesn't work in titles sadly, but I'm fine with caps NOT. I had it like that at one point.